setup.py

```diff
@@ -35,6 +35,10 @@
         'alpaca_trade_api',
         'alpaca_trade_api.polygon',
     ],
+    package_data={
+        'alpaca_trade_api': ['py.typed'],
+        'alpaca_trade_api.polygon': ['py.typed'],
```

Comment:
Hi Blane-Nelson, thanks for contributing!

Why have you added a `py.typed` file for `alpaca_trade_api.polygon` package? What is the purpose?

Reply:
This file lets utilities (like MyPy) know that the package is using the new typing library. The files doesn't need to contain anything, it just has to be present in the package's root folder.  Refer to the  PEP 651 - https://www.python.org/dev/peps/pep-0561/ on python.org for a full explanation.